docs/components/checkboxes.md

```diff
@@ -51,37 +51,58 @@ usage: >-
   <h3>Use cases</h3>
 
 
-  TBD.
+  * Use checkboxes when the user can select more than one option from a list. 
+
+  * Use [radio buttons](/components/radio-buttons) when the user can choose only
```

Comment:
`radio buttons` link 404s in the preview, not sure if this will be the case when this PR is merged?